workshop/Lab3/README.md

```diff
@@ -0,0 +1,291 @@
+# Lab 3: Scale and update apps natively, building multi-tier applications.
+
+In this lab you'll learn how to deploy the same guestbook application we deployed
+in the previous labs, however, instead of using the `kubectl` command line helper
+functions we'll be deploying the application using configuration files. The configuration
+file mechanism allows you to have more fine-grained control over all of resources
+being created within the Kubernetes cluster.
+
+Before we work with the application we need to clone a github repo from which we can get 
+all our configuration files.
+
+``` 
+	$git clone git@github.com:IBM/guestbook.git
+```
```

Comment:
You should tell people to 'cd' into 'guestbook'

Reply:
done